tests/common.py

```diff
@@ -230,6 +231,9 @@ def compare_forces(
         params = (params.astype(np.float32)).astype(np.float64)
         assert params.dtype == np.float64
         ref_u = ref_potential(x, params, box)
+        assert (
+            np.abs(ref_u) < np.iinfo(np.int64).max / FIXED_EXPONENT
+        ), "System is invalid, GPU platform unable to represent energies"
```

Comment:
nit: seems like it would be nice to import `FIXED_EXPONENT` from `custom_ops`, to avoid having to keep Python and C++ definitions in sync. Would it be possible to export this from the wrapper?

Reply:
Will make a case to do this in a follow-up PR

Reply:
+1 (later pass)